crates/exec-wasmtime/src/runtime/identity/mod.rs

```diff
@@ -74,28 +79,27 @@ fn csr(pki: &PrivateKeyInfo<'_>, exts: Vec<Extension<'_>>) -> anyhow::Result<Vec
     let signature = BitStringRef::from_bytes(sig.as_ref()).context("failed to parse signature")?;
     CertReq {
         info,
-        algorithm,
-        signature,
+        algorithm: algorithm.ref_to_owned(),
+        signature: signature.ref_to_owned(),
     }
-    .to_vec()
+    .to_der()
     .context("failed to encode CSR")
 }
 
 /// Generates a new private key and corresponding CSR
 #[instrument]
 pub fn generate() -> anyhow::Result<(Zeroizing<Vec<u8>>, Vec<u8>)> {
-    let platform = Platform::get().context("failed to query platform")?;
+    let platform = Platform::get().unwrap(); //.context("failed to query platform")?;
```

Comment:
why?

Reply:
Sorry, this was left over from me debugging some crypto issues.